nodes/nomos-node/src/config.rs

```diff
@@ -264,4 +270,15 @@ impl Config {
 
         Ok(self)
     }
+
+    pub fn update_da(mut self, da_args: DaArgs) -> Result<Self> {
+        let DaArgs { da_voter_key } = da_args;
+
+        if let Some(private_key) = da_voter_key {
+            let bytes = <[u8; 32]>::from_hex(private_key)?;
+            self.da.da_protocol.voter = bytes;
+        }
```

Comment:
Thank you. I forgot to add this function here.

I think it might be clearer to not call this `private key`, since we don't know yet what will be used for the `voter`, as discussed in https://github.com/logos-co/nomos-node/pull/498#discussion_r1378741169 (even though we'll anyway use the same private key as the consensus service for now).

Reply:
Thanks, that makes sense! Updated.